beta/tests/tensorflow/sparsity/test_common.py

```diff
@@ -122,7 +133,6 @@ def run_epoch_with_per_step_sparsity_check(steps_per_epoch: int, scheduler: Poly
                                                epoch):
         assert len(ref_vals) == steps_per_epoch
         scheduler.epoch_step(epoch if explicit else None)
-        set_sparsity_mock.assert_not_called()
```

Comment:
It should be reverted if https://github.com/openvinotoolkit/nncf/pull/560 will be merged